TypeCobol/Compiler/Nodes/Attributes.cs

```diff
@@ -56,6 +56,7 @@ public static class Attributes
             attributes["global"] = new GlobalAttribute();
             attributes["useglobalstoragevariable"] = new UseGlobalStorageVariableAttribute();
             attributes["sourceprogramhash"] = new SourceProgramHashAttribute();
+            attributes["isboolsettofalse"] = new IsBoolSetToFalseAttribute();
```

Comment:
The name is not very clear, why not `istypecobolsettofalse` (and matching attribute and flag).

Reply:
+1 for name `istypecobolsettofalse`.

Reply:
Done